docs/src/main/sphinx/release/release-400.md

```diff
@@ -4,6 +4,8 @@
 
 * Add output buffer utilization to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14396`)
 * Increase concurrency for large clusters. ({issue}`14395`)
+* Fix JSON serialization failure for `QueryCompletedEvent` in event listener.
```

Comment:
Added to General section since we never had an explicit Event listener section in the past .. maybe we also need to add something to the SPI section?